SwiftyJSONAccelerator/SJEditorViewController.swift

```diff
@@ -110,6 +121,30 @@ class SJEditorViewController: NSViewController, NSTextViewDelegate {
         }
     }
 
+    @IBAction func recalcEnabledBoxes(sender: AnyObject) {
+        
+        let supportSwiftyState = self.supportSwiftyJSONCheckbox?.state == 1 ? true : false
+        let supportObjectMapperState = self.supportObjectMapperCheckbox?.state == 1 ? true : false
+
+        if supportSwiftyState
+        {
```

Comment:
Can we have the `{` in the same line as the `if` just to be consistent throughout the project. If  it is too much don't worry about it I will fix it in the next commit.